### DISCUSSION TITLE
Remove unsigned datatypes

### DISCUSSION THREAD

**Discussion by sundy-li** [databendlabs/databend#4926] [General]
Most databases did not have unsigned data types, unsigned is just a constraint to validate the values.

Proposal:

1. Remove all unsigned datatypes in datatype implementation
2. Add constraint in column's meta
3. TransformConstriant

**Comment by bohutang** [comment]
Remove the unsigned data types from doc: https://github.com/datafuselabs/databend/pull/4930
